### ISSUE TITLE
CodeSandbox example errors

### ISSUE THREAD

**Issue by PeterC66** [Contiamo/react-connect-context#4]
# Summary

When opening the Sandbox I get TypeError: Cannot set property '_currentValue' of undefined

## Steps to Reproduce the Issue:

Click on link under Full Example on [react-connect-context](https://github.com/Contiamo/react-connect-context)

## Proposed Solution:

I am fairly new to React so don't know where to start.

(I like the look of what you have done and plan to try using it.)

**Comment by TejasQ** [comment]
Hi Peter! Thank you so much for opening this issue!

There's actually a [really easy way](https://reactjs.org/docs/context.html#accessing-context-in-lifecycle-methods) to do what this library does using plain old React out of the box and so I'd recommend taking a look [over there at the Context docs](https://reactjs.org/docs/context.html#accessing-context-in-lifecycle-methods).

Since this is so easy to do with just React, we're not sure how to proceed with this library and are currently discussing deprecating it and just pointing folks to React's amazing documentation instead.

Let me know if I can help you use it. If we decide to deprecate this library, we'll for sure add a CodeSandbox link teaching folks how to achieve the same behavior within React.

[Here's that link for now](https://codesandbox.io/s/jvl5pp1o4y).

As an aside, **it is great to hear you're new to React!** Have you familiarized yourself with other concepts/APIs already? If not, I'd suggest looking at Context at a much later stage as it's something that you might not have to use right away.

**Comment by PeterC66** [comment]
Thanks, that is good to know.  I mentioned being new to React (and lots of other things like github, npm etc), but I have been trying it out with Redux (and MongoDB in the background) for a month now - and getting things working OK after lots of looking things up!

I realised that one bit of my data would be much better handled via Context, so I looked for the best way to approach it. With your encouragement I shall explore using just plain React for the Context.
